chat.iamb.iamb.metainfo.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>chat.iamb.iamb</id>
+
+  <name>iamb</name>
+  <summary>A terminal Matrix client for Vim addicts</summary>
+  <url type="homepage">https://iamb.chat</url>
+
+  <releases>
+    <release version="0.0.9" date="2024-03-28"/>
+  </releases>
+
+  <developer_name>Ulyssa</developer_name>
+  <update_contact>git@ulyssa.dev</update_contact>
+
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://iamb.chat/static/images/iamb-demo.gif</image>
+      <caption>Example conversation within iamb</caption>
+    </screenshot>
+  </screenshots>
+
+  <description>
+    <p>
+      iamb is a client for the Matrix communication protocol.  It provides a terminal user interface with familiar Vim keybindings, and includes support for multiple profiles, threads, spaces, notifications, reactions, custom keybindings, and more.
+    </p>
+  </description>
+
+  <icon type="stock">chat.iamb.iamb</icon>
```

Comment:
```suggestion
```

don't use stock icons.

Reply:
I've replaced this with:

```
  <icon type="remote">https://iamb.chat/images/iamb.svg</icon>
```

(The `stock` icon was what the MetaInfo generator had selected when I put in that it's a CLI app)

Reply:
this is worse. just don't put one.

Reply:
actually since this is console application, you do need to specify a stock icon. Otherwise the Flathub page won't show an icon.

Icons aren't picked from desktop files for console apps since desktop files are not guaranteed to exist for console apps.

If this was a desktop application, then using stock icon would be a bug.

Reply:
I tried to go back to the `stock` icon, but removing `appstream-compose: false` breaks that. I've removed the `icon` tag, but I think it should be okay since there is an `iamb.desktop` file here.

Reply:
Removing `appstream-compose: false` is not an option.

Reply:
You told me to do so [here](https://github.com/flathub/flathub/pull/5110#discussion_r1725680963)?

Reply:
Should I add it back?

Reply:
That's probably a typo. Don't use `appstream-compose: false`

The issue with the icon tag is, it should be fixed instead.

Reply:
Icon seems fine.